### DISCUSSION TITLE
DeploymentError: more than 5000, code: "api-upload-free" on PRO plan

### DISCUSSION THREAD

**Discussion by hynra** [vercel/vercel#4348] [General] [answered]
I don't know if it's relevant if I post here because there is no further reply from the ticket that I sent to the support section. It seems like it's quite active here, so I decided to post it here.
Okay, so I initially tried the deployment feature via Vercel CLI on a personal account using the Hobby plan. Everything went well until deployment was hit upload limitations (on the pricing page, upload limitations were 5000 per day).

```
DeploymentError: Too many requests - try again in 20 hours (more than 5000, code: "api-upload-free")
```
So to overcome this limitation, I tried upgrading to the Pro plan, then made a team of myself. Then I tried re-deployment using the Vercel CLI by using the scope team that I created earlier, everything went smoothly via the CLI. So in the scope of the team, there is one project that has been made. It seems enough to try the deployment feature via Vercel CLI.
It's time to try importing the project via my Github Organization. Integration was successful, I tried to select a repository to import, but I got the same limitation as to
 when using a personal account, even though I imported it in the scope team.

![tempsnip](https://user-images.githubusercontent.com/3982650/81636482-2eaa0900-943e-11ea-8ea4-dbc27c29ab40.png)

**Comment by leo** [comment, marked as answer]
Thank you for sharing! Can you try [this import flow](https://vercel.com/import2) and tell me if it works for you? We've defined the way files are uploaded.
